aks/extra_variables.tf

```diff
@@ -7,3 +7,8 @@ variable "dns_zone_resource_group_name" {
   description = "The Azure DNS zone's resource group name."
   type        = string
 }
+
+variable "oidc_issuer_url" {
```

Comment:
This variable is usually called `cluster_oidc_issuer_url` in several modules (maybe used only on EKS?). I think it would make sense to have the same name everywhere (unless it has a different meaning).

Reply:
C.f. https://github.com/camptocamp/devops-stack-module-cert-manager/blob/main/eks/extra_variables.tf#L1

Reply:
I agree, done.